cps/kobo.py

```diff
@@ -535,16 +535,16 @@ def get_current_bookmark_response(current_bookmark):
     return resp
 
 
-@kobo.route("/<book_uuid>/image.jpg")
+@kobo.route("/<book_uuid>/<width>/<height>/image.jpg")
 @requires_kobo_auth
```

Comment:
I added the unused "Quality" and "isGreyscale" paths to the route, so we are nearer to Kobo's image url and this causes less trouble in the future